css/src/tokens/schemes.scss

```diff
@@ -0,0 +1,5 @@
+$schemes: (
```

Comment:
I think an explanatory comment here would be helpful here. One saying that this object's keys should match themes and that it's value will set the color-scheme css property for that theme.

Reply:
Updated